src/transformers/integrations.py

```diff
@@ -199,8 +199,9 @@ def _objective(trial, local_trainer, checkpoint_dir=None):
         trainer.use_tune_checkpoints = True
         if kwargs["keep_checkpoints_num"] > 1:
             logger.warning(
-                "Currently keeping {} checkpoints for each trial. Checkpoints are usually huge, "
-                "consider setting `keep_checkpoints_num=1`.".format(kwargs["keep_checkpoints_num"])
+                f"Currently keeping {kwargs['keep_checkpoint_num']} checkpoints for each trial. "
+                f"Checkpoints are usually huge, "
+                "consider setting `keep_checkpoints_num=1`."
```

Comment:
Not sure this will pass quality -> isort usually complains about f-strings without any {}
```suggestion
                "Checkpoints are usually huge, "
                "consider setting `keep_checkpoints_num=1`."
```

Reply:
Fixed!